stroeer/core/v1/article.proto

```diff
@@ -193,6 +193,12 @@ message Article {
     COMPLIANCE = 12;
     // _Recipe_ — a cooking recipe article
     RECIPE = 13;
+    // _Report_ — a report article
+    REPORT = 14;
+    // _Analyses_ — a detailed analysis article
+    ANALYSES = 15;
+    // _Guest Article_ — a guest article, usually written by an external author
+    GUEST_ARTICLE = 16;
```

Comment:
We never used the postfix `ARTICLE` before. Maybe only `GUEST` is enough?

Reply:
GUEST_OPINION
wenn es sich fast immer um Meinungsbeiträge handelt

Reply:
Den, den die seriöse NY Times nutzt: `GUEST_ESSAY`. `GUEST_OPINION` klingt nach falschem englisch, jedoch nutzt die Washington Post den: https://www.washingtonpost.com/opinions/guest-opinions/

Geht beides klar, zur Not wirf 'ne Münze.